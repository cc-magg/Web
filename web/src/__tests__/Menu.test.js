import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import MenuContainer from '../components/menu/container/menu'
import MenuComponent from '../components/menu/component/menu'

test('Menu should render as expected', () => {
    const wrapper = shallow(<MenuContainer />);
    const tree = toJson(wrapper);

    expect(tree).toMatchSnapshot();
});

test('MenuContainer should render as expected', () => {
    const wrapper = shallow(<MenuComponent />);
    const tree = toJson(wrapper);

    expect(tree).toMatchSnapshot();
});
