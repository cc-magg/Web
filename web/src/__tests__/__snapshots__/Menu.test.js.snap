// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render as expected 1`] = `<_default />`;

exports[`MenuContainer should render as expected 1`] = `
<div
  className="jsx-186130977"
>
  <nav
    className="jsx-186130977 navbar navbar-expand-lg navbar-dark"
  >
    <a
      className="jsx-186130977 navbar-brand"
      href="#"
    >
      MAGG
    </a>
    <button
      aria-controls="navbarTogglerDemo02"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="jsx-186130977 navbar-toggler"
      data-target="#navbarTogglerDemo02"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="jsx-186130977 navbar-toggler-icon"
      />
    </button>
    <div
      className="jsx-186130977 collapse navbar-collapse flex-row-reverse"
      id="navbarTogglerDemo02"
    >
      <ul
        className="jsx-186130977 navbar-nav mr-0 mt-2 mt-lg-0"
      >
        <li
          className="jsx-186130977 nav-item active"
        >
          <a
            className="jsx-186130977 nav-link"
            href="#"
          >
            Home 
            <span
              className="jsx-186130977 sr-only"
            >
              (current)
            </span>
          </a>
        </li>
        <li
          className="jsx-186130977 nav-item"
        >
          <a
            className="jsx-186130977 nav-link"
            href="#"
          >
            Link
          </a>
        </li>
        <li
          className="jsx-186130977 nav-item"
        >
          <a
            aria-disabled="true"
            className="jsx-186130977 nav-link disabled"
            href="#"
            tabIndex="-1"
          >
            Disabled
          </a>
        </li>
      </ul>
    </div>
  </nav>
  <JSXStyle
    id="186130977"
  >
    nav.jsx-186130977{background-color:#353590;}
  </JSXStyle>
</div>
`;
